cocotb/decorators.py

```diff
@@ -510,7 +510,8 @@ async def my_test(dut):
             .. versionchanged:: 1.3
                 Specific exception types can be expected
         skip (bool, optional):
-            Don't execute this test as part of the regression.
+            Don't execute this test as part of the regression. Test can still be ran
+            manually by setting :make:var:`TESTCASE`.
```

Comment:
```suggestion
            Don't execute this test as part of the regression. Test can still be run
            manually by setting :make:var:`TESTCASE`.
```
...since this is not simple past but, uh, past participle?

Reply:
I believe you're correct; sorry about that. I've fixed this here and in (I think) the other places and rebased.